packages/@unimodules/react-native-adapter/scripts/ios/autolinking_manager.rb

```diff
@@ -118,7 +118,7 @@ class AutolinkingManager
       ]
 
       if !search_paths.nil? && !search_paths.empty?
-        args.concat(searchPaths)
+        args.concat(search_paths)
```

Comment:
🤔

Reply:
`searchPaths` was actually wrong from the beginning 😅 looks like I didn't test this case when I was writing this script 🙈